Formula/csound.rb

```diff
@@ -5,8 +5,8 @@ class Csound < Formula
       tag:      "6.17.0",
       revision: "f5b4258794a82c99f7d85f1807c6638f2e80ccac"
   license "LGPL-2.1-or-later"
-  revision 5
-  head "https://github.com/csound/csound.git", branch: "develop"
+  revision 6
```

Comment:
Are we actually changing anything about how this build on macOS?  If not, I don't think we need to revision bump this.  I'll just dispatch a bottle build once we merge the changes.

Reply:
We’re not; I reverted the revision to 5.